_posts/2025-05-21-cockpit-339.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Cockpit 339
+author: tmatus
+date: '2025-05-21'
+tags: cockpit, podman, files, machines, ostree
+slug: cockpit-339
+category: release
+summary: 'Cockpit-ws arm64 container, warn on SMART failures, Cockpit no longer modifies the "shareable" attribute of disks'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 339, cockpit-podman 106,
+cockpit-files 21, cockpit-machines 332, and cockpit-ostree 210:
+
+## Add cockpit/ws arm64 container
+
+The [cockpit webserver container image](https://quay.io/repository/cockpit/ws?tab=info) is now available for the arm64 architecture as well.
+
+_Thanks to Marvin Ruder for this contribution!_
+
+## Storage: Disk Self-Test error warnings
+
+The health card on the overview page now provides proactive warnings when a disk reports errors from its SMART self-test.
+
+![smart]({{ site.baseurl }}/images/339-overview-smart-warning.png)
+
+## Machines: The "shareable" attribute of disks is no longer modified by Cockpit
+
+By default, libvirt can attach a disk to multiple virtual machines, but if the disk allows writing, only a single one of these machines
+can be running at a given time. Writing to a disk concurrently from multiple VMs at the same time can easily lead to data corruption,
+and libvirt prevents this unless the "shareable" attribute of a disk is set. Previously, Cockpit would set this attribute without
+much fuzz and would thus invite data corruption. To avoid this, Cockpit no longer mark disks as "shareable" when attaching them to multiple VMs.
```

Comment:
"Without much fuzz" i think you mean "without much fuss", but maybe we should be specific here and say something like "by default".

Reply:
also there should be: 
Cockpit no longer **marks** disks as "shareable"